src/frontends/paddle/tests/test_models/gen_scripts/generate_set_value.py

```diff
@@ -0,0 +1,148 @@
+
+
+def build_slice(starts, ends, steps) -> list:
+    outs = []
+    for st, ed, step in zip(starts, ends, steps):
+        outs.append(slice(st, ed, step))
+    return outs
+
+
+def generate_data(data, dtype):
+    return [np.array(d).astype(dtype) for d in data]
+
+
+def main():
+    shape = (2, 3, 4, 5)
+    dtype = "float32"
+    data = np.random.random(shape).astype(dtype)
+    value = np.array([0]).astype(dtype)
+
+    def set_value1(x, value):
+        x[1:2, ::2, 2:4] = value
+        return x
+
+    paddle_set_value("set_value1", data, value, set_value1, dtype)
+
+    shape = (5, 3, 1)
+    dtype = "float32"
+    data = np.random.random(shape).astype("float32")
+    value = np.random.random((3, 3, 1)).astype(dtype)
+
+    def set_value2(x, value):
+        x[2:5] = value
+        return x
+
+    paddle_set_value("set_value2", data, value, set_value2, dtype)
+
+    shape = (10, 2, 5)
+    dtype = "int32"
+    data = np.random.randint(0, 5, shape).astype(dtype)
+    value = np.random.randint(0, 2, (10, 2, 3)).astype(dtype)
+
+    def set_value3(x, value):
+        x[:, :, -4:-1] = value
+        return x
+
+    paddle_set_value("set_value3", data, value, set_value3, dtype)
+
+    shape = (10, 2, 5)
+    dtype = "float32"
+    data = np.random.randn(*shape).astype(dtype)
+    value = np.random.randn(3, 1, 1).astype(dtype)
+    starts = generate_data([-4, 0, 1], np.int64)
+    ends = generate_data([-1, 1, 3], np.int64)
+    steps = generate_data([1, 1, 1], np.int64)
+
+    def set_value4(x, value, *slice):
+        x[build_slice(*slice)] = value
+        return x
+
+    paddle_set_value("set_value4", data, value, set_value4, dtype, starts, ends, steps)
+
+    shape = (10, 5)
+    dtype = "int32"
+    data = np.random.randint(0, 5, shape).astype(dtype)
+    value = np.random.randint(0, 2, (1, )).astype(dtype)
+    starts = generate_data([-4], np.int64)
+    ends = generate_data([-1], np.int64)
+    steps = generate_data([1], np.int64)
+
+    def set_value5(x, value, *slice):
+        x[build_slice(*slice)] = value
+        return x
+
+    paddle_set_value("set_value5", data, value, set_value5, dtype, starts, ends, steps)
+
+    shape = (10, 5)
+    dtype = "float32"
+    data = np.random.randint(0, 5, shape).astype(dtype)
+    value = np.random.randint(0, 2, (10, 3)).astype(dtype)
+
+    def set_value6(x, value):
+        x[:, -4:-1] = value
+        return x
+
+    paddle_set_value("set_value_dynamic1", data, value, set_value6, dtype, is_dynamic=True)
+
+    shape = (10, 5)
+    dtype = "int32"
+    data = np.random.randint(0, 5, shape).astype(dtype)
+    value = np.random.randint(0, 2, (1, )).astype(dtype)
+    starts = generate_data([-4], np.int64)
+    ends = generate_data([-1], np.int64)
+    steps = generate_data([1], np.int64)
+
+    def set_value7(x, value, *slice):
+        x[build_slice(*slice)] = value
+        return x
+
```

Comment:
Can you add some test case with step > 1?

Reply:
done, but I can't test since there are some issue about `StridedSlice` 
![2023-04-19_21-40](https://user-images.githubusercontent.com/72954905/233093744-93ee347e-991a-4f1d-aecd-cbda5c31e607.png)